website/docs/r/dns_linked_zone.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+subcategory: "DNS Services"
+layout: "ibm"
+page_title: "IBM : dns_linked_zone"
+description: |-
+  Manages IBM Private DNS linked zone.
+---
+
+# ibm_dns_linked_zone
+
+The DNS linked zone resource allows users to request and manage linked zones. 
+
+
+## Example usage
+
+```
+resource "ibm_resource_instance" "test-pdns-instance" {
+  name              = "test-pdns"
+  resource_group_id = data.ibm_resource_group.rg.id
+  location          = "global"
+  service           = "dns-svcs"
+  plan              = "standard-dns"
+}
+
+resource "ibm_dns_zone" "test-pdns-zone" {
+  name        = "test.com"
+  instance_id = ibm_resource_instance.test-pdns-instance.guid
+  description = "testdescription"
+  label       = "testlabel-updated"
+}
+
+resource "ibm_dns_linked_zone" "test" {
+  name          = "test_dns_linked_zone"
+  instance_id = ibm_resource_instance.test-pdns-instance.guid
+  description   = "seczone terraform plugin test"
+  owner_instance_id = "**************************"
```

Comment:
Instead of "***" Can you describe 

owner_instance_id = "OWNER Instance ID"
 owner_zone_id = "OWNER ZONE ID"